qutebrowser/javascript/history.js

```diff
@@ -0,0 +1,194 @@
+/**
+ * Copyright 2017 Imran Sobir
+ *
+ * This file is part of qutebrowser.
+ *
+ * qutebrowser is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * qutebrowser is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with qutebrowser.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+"use strict";
+
+window.loadHistory = (function() {
+    // The time of last history item.
+    var lastTime = null;
```

Comment:
The name (+ desc) is pretty meaningless.
Also I'd rather save the whole date, why would you cut the milliseconds? You just have to use the "magic number" 1000 everywhere.